core/src/main/java/com/scalar/db/transaction/consensuscommit/ConsensusCommitAdmin.java

```diff
@@ -165,6 +177,9 @@ public void addNewColumnToTable(
       String namespace, String table, String columnName, DataType columnType)
       throws ExecutionException {
     TableMetadata tableMetadata = getTableMetadata(namespace, table);
+    if (tableMetadata == null) {
+      throw new IllegalArgumentException("Table does not exist: " + namespace + "." + table);
```

Comment:
```suggestion
      throw new IllegalArgumentException("Table does not exist: " + ScalarDbUtils.getFullTableName(namespace, table));
```
Likewise.

Reply:
Fixed in fbe71953b4d40af8bb7e5ad1be662096ff7b164c. Thanks.